### ISSUE TITLE
License Infringement

### ISSUE THREAD

**Issue by proudparrot2** [tgcofficial/tgcofficial.github.io#1]
Hey there!

I help out with [3kh0.github.io](https://github.com/3kh0/3kh0.github.io), and I’ve noticed that your website is almost an exact copy of his.

Our repository and website is protected with a Creative Commons No Derivatives license which prohibits the modification of our website. Please take this website down, as it violates our license, and your repository may be subject to a DMCA takedown via GitHub.

If you have any questions, you can DM me on Discord at **proudparrot2#2222* or join our Discord server at https://discord.gg/3kh0

Thanks!
proudparrot2

**Comment by proudparrot2** [comment]
If you want to use the games, you can create your own website and ask 3kh0 for permission.

**Comment by Asleep123** [comment]
^ You have 48 hours to comply with this takedown request. If you do not comply, further action will be taken upon your repository.

**Comment by Rahib777-7** [comment]
we assk ghothoob to help us

**Comment by K74K3N** [comment]
This is freakin' hilarious. You realize this is some kid right? Oh man, I love the internet. I wish the internet would use my countries laws, "If you are an unrecognized business (not official business), no legal action can be taken." But, GitHub seems to love 10 year old drama. If you didn't want people editing your code, you should have not chosen GitHub. Honestly.

**Comment by K74K3N** [comment]
#By setting your repositories to be viewed publicly, you agree to allow others to view and fork your repositories.

That's github.

**Comment by Goob-r** [comment]
I mean it is in the copyright does say that they couldn't do that, but also it's GitHub so why would you post your code here if you didn't want it to be used

**Comment by K74K3N** [comment]
The purpose of forking is to edit the code, or to push other code onto the original repo.

**Comment by K74K3N** [comment]
> Our repository and website is protected with a Creative Commons No Derivatives license which prohibits the modification of our website. Please take this website down, as it violates our license, and your repository may be subject to a DMCA takedown via GitHub.

Where's that licence? I can't find it.

>If you want to use the games, you can create your own website and ask 3kh0 for permission.

As much as I might respect the guy, it's not like 3kh0 individually coded all of those games himself. They are owned by other companies, and they could just as easily take down your site as well.

>You have 48 hours to comply with this takedown request. If you do not comply, further action will be taken upon your repository.

He can just move it to his own repository, or download it and load it on his own server. This means nothing. No one cares.

**Comment by Requals** [comment]
watch out 3kh0 has a group of goons that do his bidding

**Comment by K74K3N** [comment]
I don't care if a bunch of school kids get mad at me.

**Comment by Asleep123** [comment]
They might not be made by 3kh0, but he did spend time ripping the game. Plus, he copied more than just the game, he copied the whole website and put his branding on it. This does violate the license terms.

**Comment by K74K3N** [comment]
Did you actually look at this kids code? I checked, and it isn't exactly the same, it can be argued this kid, took "inspiration" from 3kh0's site.

**Comment by proudparrot2** [comment]
So by inspiration you mean copying the background, navbar, and button animations as 3kh0?

**Comment by K74K3N** [comment]
Scroll down on the home, it is not the same background as 3kh0. The navigational bar, is typical for sites, look for example at GitHub's navbar, it is similar. Look at both Reddit and Bandlab's navigation bars. Also, Button animations are just bashful to get mad about.

**Comment by proudparrot2** [comment]
We agree that yes he did not code each game, I asked him why we need to put credit and he said it took time to gather them for his website which takes effort and storage. The games aren’t a huge problem at the moment.

My problem is he took the theme and general site feel and layout as our site, took credit for it, and didn’t even correctly credit the right people. Nebelung#1335 made almost all of the v4 website, but there are no credits for him either.

We are about this because the developers took time out of their lives to make this content for people, and it’s not appreciated when their work and design is taken and claimed as other people’s. If you have any other questions, please join our Discord, we can answer questions there.

**Comment by proudparrot2** [comment]
When you initially load into TGC, it’s almost identical to the initial loading of 3kh0. Tell me I’m wrong.

**Comment by proudparrot2** [comment]
The reason I’m talking about button animations means it’s more similar to 3kh0, which is more reason to believe that it’s copied from us.

**Comment by proudparrot2** [comment]
> #By setting your repositories to be viewed publicly, you agree to allow others to view and fork your repositories.
> 
> That's github.

I’d like to point out that this is not in fact true, there are rules for open source software and you do have to comply with the licenses, because it’s the developers work and they can choose how people use and manage their content.

That’s GitHub and all of open source software.

**Comment by Rahib777-7** [comment]
> When you initially load into TGC, it’s almost identical to the initial loading of 3kh0. Tell me I’m wrong.

no cuz it not load

**Comment by proudparrot2** [comment]
Yeah that’s true, something’s really weird on mobile devices - it only loads and works on laptops…

**Comment by K74K3N** [comment]
Truly, I really don't care, you bring up some good points. I will stop. This time.

**Comment by tgcofficial** [comment]
Hello sorry for the late response. I asked 3kh0 if I can use his assets and he said yes. I have uploaded some of my own games to his site.

**Comment by tgcofficial** [comment]
+ the home page

**Comment by tgcofficial** [comment]
Please contact 3kh0 before contacting me.

- saturn

**Comment by tgcofficial** [comment]
I agree with K74K3N because all the games were site ripped using the network tab, the games are owned by different companies. For example "Soccer Skills Euro Cup". 3kh0 or one of the devs added their watermark to the game kinda acting like it's their game.

**Comment by tgcofficial** [comment]
Talk to me via discord saturn#0020

**Comment by tgcofficial** [comment]
Oh yeah plus on credits I have 3kh0 on credits for navbar. Its been up there for a while. I never called it mine.

**Comment by tgcofficial** [comment]
![image](https://user-images.githubusercontent.com/123127031/228911617-e6285d77-5309-4e4d-9aeb-1b8172e3c517.png)

**Comment by tgcofficial** [comment]
Yeah thats what I thought

**Comment by ThatOnePers0n** [comment]
> Oh yeah plus on credits I have 3kh0 on credits for navbar. Its been up there for a while. I never called it mine.

I noticed this & notified the staff, however, nobody else noticed

**Comment by tgcofficial** [comment]
Thanks!

**Comment by Rahib777-7** [comment]
https://user-images.githubusercontent.com/74979271/228953222-fc1b23c5-3bd0-4c1f-ae4c-245ea05799f9.MOV

**Comment by ThatOnePers0n** [comment]
> Anyways I made sure before I added stuff from 3kh0 I always asked him.

Thanks for being so good with this, it was a mistake made by staff, you can probably close the ticket now!

**Comment by tgcofficial** [comment]
Sounds good
